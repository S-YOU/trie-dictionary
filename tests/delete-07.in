insert aa
insert ab
find ab
delete 0
find ab
