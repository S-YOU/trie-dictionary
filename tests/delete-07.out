word number: 0
word number: 1
YES
deleted: 0
YES
